core/controllers/acl_decorators.py

```diff
@@ -227,11 +227,11 @@ def can_view_skills(handler):
         can view multiple given skills.
     """
 
-    def test_can_view(self, comma_separated_skill_ids, **kwargs):
+    def test_can_view(self, selected_skill_ids, **kwargs):
```

Comment:
Is this decorator used in handlers that already have the schema defined? If yes then I think we a remove the lines 248–252

Reply:
Sorry, I mean `can_view_skills` is only used in `ConceptCardDataHandler`, can we check the format of skill IDs there and then we can remove lines 248–252?

Reply:
No, I think since `selected_skills_id` is a url path element so even if we give `invalid_skill_id`, the handler code will never get executed because we will receive `404 page not found error` from the decorator itself. So I think format validation part must be handled in the decorator.

Reply:
No, I think since `selected_skills_id` is a url path element so even if we give `invalid_skill_id`, the handler code will never get executed because we will receive `404 page not found error` from the decorator itself. So I think format validation part of skill_ids must be handled in the decorator.

Reply:
Ah, right, good point, okay lets keep it as is.